image_server/cloudformation.yml

```diff
@@ -43,13 +43,6 @@ Resources:
           SourceSecurityGroupId: !Ref LoadBalancerSecurityGroup
           ToPort: 6443
           FromPort: 6443
-      VpcId: !Ref VpcId
-
-  SshSecurityGroup:
-    Type: AWS::EC2::SecurityGroup
-    Properties:
-      GroupDescription: !Sub "Security group for ${AWS::StackName} servers"
-      SecurityGroupIngress:
         - IpProtocol: tcp
```

Comment:
Does the image server need both types of IP protocols? I seem to recall this being the case (and writing these very words a few times before), but wasn't sure if that was still the case for the new partitioned approach.